js/index.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable no-use-before-define */
+/* global Book, createElement */
+
+class BookCollection {
+  constructor() {
+    this.books = [];
+    this.populate();
+  }
+
+  // Populate books section dynamically using local storage
+  // Called from page load, from adding a book, and from removing a book
+  populate() {
+    const bookList = document.querySelector('.book-list');
+    bookList.innerHTML = '';
+
+    if (localStorage.length > 0) {
+      const booksLS = JSON.parse(localStorage.getItem('books'));
+
+      this.books = [];
+
+      booksLS.forEach((book, index) => {
+        this.books.push(new Book(index + 1, book.title, book.author));
+
+        const bookContainer = document.createElement('div');
+        const bookTitle = createElement('p', '', {}, book.title);
+        const bookAuthor = createElement('p', '', {}, book.author);
+        const separator = document.createElement('hr');
+
+        const removeButton = createElement('button', 'remove-btn', {}, 'Remove');
+        removeButton.addEventListener('click', (e) => {
+          bookCollection.remove(e);
+        });
+        bookContainer.append(bookTitle, bookAuthor, removeButton, separator);
+        bookList.append(bookContainer);
+      });
+    }
+  }
+
+  // Add book to collection
+  add() {
+    this.books.push(new Book(
+      this.books.length + 1,
+      document.getElementById('title').value,
+      document.getElementById('author').value,
+    ));
+
+    this.refresh();
+  }
```

Comment:
- [ ] Please add `input values` outside the class. Let's try to fix this issue by adding the `title` and `author` parameters in our `add` method. After adding books this `add` method should return the `books` property or by adding a new getter method. And pass the input values on the `add` method after creating a new object.